### DISCUSSION TITLE
ARCropolis Hanging While Loading Super Smash Bros. Ultimate Replays

### DISCUSSION THREAD

**Discussion by PikaRed25** [Raytwo/ARCropolis#568] [Issues]
I'm currently using the latest version of the Nintendo Switch System Firmware, Super Smash Bros. Ultimate, Atmosphere, and ARCropolis. Whenever I try to load a replay saved from version 13.0.1 and prior of Super Smash Bros. Ultimate, the game hangs and never loads the replay. From what I've found, the game is able to load replays saved from version 13.0.2 as well as load all replays saved from all versions on a vanilla boot with no issues.

**Comment by Xalavar** [comment]
Same issue with replays not loading. Updating Skyline didn't do anything either and this issue appears to be exclusive to Yuzu for me. (I haven't tried Ryujinx.)

My only option is reverting back to 4.0.1.

**Reply by McWTFMan** [reply]
I have the same issue as well. Wouldn't reverting back to 4.0.1 not fix this issue since it doesn't support the latest version of Smash Ultimate?

**Reply by Xalavar** [reply]
I should have been more specific. I meant I was having this issue on an emulator. My only resolution was to just stay on SSBU update 13.0.1 while using the emulator.

**Reply by McWTFMan** [reply]
I see. I'm still having that issue on official hardware tho. Guess I'll have to wait for the next update to fix this.

**Reply by Xalavar** [reply]
Actually, disregard what I said about it running fine on official hardware. It's the same situation: only loads replays from Update 13.0.2 and all other replays result in a black screen.

**Reply by McWTFMan** [reply]
I see. At least it's not just me that's having the issue. I guess the only way to view the replays is by holding L while launching Ultimate. However, I imagine it would only work for replays that don't involve any moveset edits or anything that impacts a vanilla match. Ex: Having up taunt to final smash on a replay won't work or play correctly when viewing it without mods, however, having Giga Bowser on your roster and using him on a match would work just fine for that replay.

**Comment by MidnightLycanr0c** [comment]
Just want to bump this thread to point out that this is still an issue, even after the 13.0.3 update.
